DetDescr/src/DetDescr/DetectorIDBindings.cxx

```diff
@@ -1,4 +1,6 @@
 #if DETECTORID_BINDINGS_ENABLED
+// This we may be able to remove after boost update
+#define BOOST_BIND_GLOBAL_PLACEHOLDERS
 #include <boost/python.hpp>
```

Comment:
The more I think about it, the less I'm convinced this is causing the change in the logs.

This `#define` is within a source file and so would not affect how the Ecal library is compiled. I guess this define would be applied (?) after the Ecal library is linked to the DetDescr library, but I have not heard of `#define` being used in this way.

Reply:
Yes I guess you are right it shouldnt be connected